backend/.env

```diff
@@ -4,9 +4,9 @@ TOKEN= ihopeitsworking
 
 # For development
 DB_HOST=localhost
-DB=iron-side-db
+DB=iron_side_db
```

Comment:
The rest of the team should be aware of this change as well, otherwise, their local instances of the DB won't work

Reply:
yes we have different local instances.. will check it for every-time.